src/library/Uno.Material/Styles/Application/v2/SharedColors.xaml

```diff
@@ -300,8 +300,8 @@
 			<SolidColorBrush x:Key="OnSurfaceDraggedBrush" Color="{StaticResource OnSurfaceColor}" Opacity="{StaticResource DraggedOpacity}" />
 			<SolidColorBrush x:Key="OnSurfaceSelectedBrush" Color="{StaticResource OnSurfaceColor}" Opacity="{StaticResource SelectedOpacity}" />
 			<SolidColorBrush x:Key="OnSurfaceMediumBrush" Color="{StaticResource OnSurfaceColor}" Opacity="{StaticResource MediumOpacity}" />
-			<SolidColorBrush x:Key="OnSurfaceLowBrush" Color="{StaticResource OnSurfaceColor}" Opacity="{StaticResource LowOpacity}" />
-			<SolidColorBrush x:Key="OnSurfaceDisabledBrush" Color="{StaticResource OnSurfaceColor}" Opacity="{StaticResource DisabledOpacity}" />
+			<SolidColorBrush x:Key="OnSurfaceLowBrush" Color="{StaticResource OnSurfaceColor}" Opacity="{StaticResource DisabledOpacity}" />
+			<SolidColorBrush x:Key="OnSurfaceDisabledBrush" Color="{StaticResource OnSurfaceColor}" Opacity="{StaticResource HoverOpacity}" />
```

Comment:
These should be kept as `LowOpacity` and `DisabledOpacity`, no?

Reply:
@kazo0 On Figma, I see that OnSurfaceDisabledBrush is set to 8% opacity which is HoverOpacity, and OnSurfaceLowBrush is set to 38% opacity which is DisabledOpacity. That is why I changed them, but I can always revert these changes if you want to keep them as is.

![image](https://github.com/unoplatform/Uno.Themes/assets/54756963/bf60ec8d-cd01-4a28-b3ac-8ce968e005f4)
![image](https://github.com/unoplatform/Uno.Themes/assets/54756963/09f8f50c-71ff-41d4-b894-4efeddf120fc)

Reply:
@NVLudwig is there a mistake in our code or yours? :)

Reply:
Is this maybe a larger problem, are all "Low" brushes supposed to be at 38% and all "Disabled" supposed to be at 8% and not just the OnSurface brushes?

If that's the case should we maybe just be changing the base values for LowOpacity and DisabledOpacity?

Reply:
@nguyeenvy for now lets leave the brushes with the LowOpacity and DisabledOpacity just so it stays consistent with naming and then we can deal with changing their values in another issue